docs/csdk-guide.md

```diff
@@ -29,6 +29,32 @@ code and without CarbonSession.
 
 In the carbon jars package, there exist a carbondata-sdk.jar, 
 including SDK reader for C++ SDK.
+
+##Compile/Build CSDK
```

Comment:
Can you add some doc for CLion? CLion also support compile/build, run, debug.

Reply:
Clion is not used for deployment of CSDK . it is IDE which is not mandatory to use CSDK . For developer, Clion, Eclipse(CDT)..etc  options are available. and developer can refer respective IDE help about how to import cmake project.(added in carbon sdk guide about this)